### PR TITLE
pip package for python library

### PR DESCRIPTION
**Pull request by RobertLucian**
Prepared python `grovepi` library for PIP packaging.
Here's an example of this repository on pypi.org: [https://pypi.python.org/pypi/grovepi-test1/0.0.5](url)

### CODE REVIEW

File: Software/Python/setup.py [hunk 1]
```diff
@@ -32,13 +32,43 @@
...
+    'Intended Audience :: Developers',
+    'Intended Audience :: Education',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: POSIX :: Linux',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 3',
+    'Topic :: Software Development :: Embedded Systems',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+    ],
+    url = "https://github.com/DexterInd/GoPiGo3",
```
Comment (DexterInd) [inline Software/Python/setup.py]: This needs to be changed to GrovePi.

File: Software/Python/setup.py [hunk 2]
```diff
@@ -32,13 +32,43 @@
...
+
+    keywords = ['robot', 'grovepi', 'grovepi+', 'dexter industries', 'learning', 'education'],
```
Comment (DexterInd) [inline Software/Python/setup.py]: Add in "Iot" and "Internet of Things" and "prototyping"

File: Software/Python/setup.py (hunk 1 above)
Comment (RobertLucian) [inline Software/Python/setup.py]: Done.

File: Software/Python/setup.py (hunk 2 above)
Comment (RobertLucian) [inline Software/Python/setup.py]: Done.

**Pull request merged**